python/pdstools/adm/ADMDatamart.py

```diff
@@ -1944,9 +1907,9 @@ def log_and_write(message: str, level: str = "INFO"):
                 check=True,
             )
             quarto_version = version_result.stdout.strip()
-            log_and_write(f"Quarto version: {quarto_version}")
+            logger.info(f"Quarto version: {quarto_version}")
         except subprocess.CalledProcessError as e:
-            log_and_write(f"WARNING: Failed to check Quarto version: {e}")
+            logger.warning(f"Failed to check Quarto version: {e}")
```

Comment:
Same as above

Reply:
Not sure if this has to be an error. Quarto version check might fail as long as the quarto process runs